docs/data/pages.ts

```diff
@@ -236,6 +233,34 @@ const pages: MuiPage[] = [
       },
     ],
   },
+  {
+    pathname: '/x/migration',
+    title: 'Migration',
+    icon: 'BookIcon',
+    children: [
+      {
+        pathname: '/x/migration',
+        subheader: 'Upgrade to v6',
+        children: [
+          { pathname: '/x/migration/migration-data-grid-v5', title: 'Breaking changes: Data Grid' },
```

Comment:
#6472 would come here

Reply:
Should be added to `docs/public/_redirects`

```
/:lang/x/react-data-grid/migration-v4/ /:lang/x/migration/migration-data-grid-v4/ 301
/:lang/x/react-data-grid/migration-v5/ /:lang/x/migration/migration-data-grid-v5/ 301
/:lang/x/react-date-pickers/migration-lab/ /:lang/x/migration/migration-pickers-lab/ 301
```